packages/framework/src/Migrations/Version20241205144230.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20241205144230 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $this->sql('CREATE EXTENSION IF NOT EXISTS pg_trgm');
```

Comment:
I just want to add that for some databases the execute user may not have sufficient permissions for this command

Reply:
so instead of adding this into a migration, we should add it to https://github.com/shopsys/shopsys/blob/16.0/packages/framework/src/Command/CreateDatabaseCommand.php, right?

Reply:
plus add an upgrade note so that the command is run after/during deploy? :thinking:

Reply:
It should be in CreateDatabaseCommand but you will probably need to migrate existing database. The problem is that if it stays here, some operators won't get through this migration and it will crash because they won't have enough permissions. So I would probably rather advise when upgrading to manually install this extension.